vim_menutrans_helper.py

```diff
@@ -0,0 +1,196 @@
+                tobe_translated_blocks = ""
+                index += 1
+                while index < len(word_list):
+                    if word_list[index] == '<silent>' or word_list[index] == '<script>' or (word_list[index][0].isdigit() and not word_list[index - 1][0].isdigit()):
+                        index += 1
+                    else:
+                        # menu_item with "ToolBar" has no following words to be translated
+                        if not ("ToolBar." in word_list[index]):
+                            tobe_translated_blocks = word_list[index]
+                        break
+
+                # string starting with sill! needs to be removed first digit
+                if len(tobe_translated_blocks) > 0 and tobe_translated_blocks[0] == '5':
+                    tobe_translated_blocks = tobe_translated_blocks[1:]
+                if len(tobe_translated_blocks) > 0 and tobe_translated_blocks[-1] == '5':
+                    tobe_translated_blocks = tobe_translated_blocks[:-1]
+
+                # menu_item with "ToolBar" constructed a tobe_translated_blocks with length equals zero
+                if len(tobe_translated_blocks) == 0:
+                    continue
+
+                # untranslated word string splitted by "."
+                tobe_translated_word = tobe_translated_blocks.split('.')
+                # untranslated word string splitted by "." but with "\." together
+                tobe_translated_words = []
+
+                # fill tobe_translated_words with '\.' together words
+                pre_backlash = False
+                for word in tobe_translated_word:
+                    if len(word) != 0 and word[0] == '-':
+                        pre_backlash = False
+                        continue
+                    if pre_backlash:
+                        tobe_translated_words[-1] += '.' + word
+                    else:
+                        tobe_translated_words.append(word)
+                    if len(word) != 0 and word[-1] == '\\':
+                        pre_backlash = True
+                    else:
+                        pre_backlash = False
+
+                # conver tobe_translated_words list to string
+                for word in tobe_translated_words:
+                    if len(word) > 0:
+                        untranslated_dict[word.lower()] = (
+                            line_number, untranslated_file)
+            elif word_list[0] in toolbar_item:
+                untranslated_dict[word_list[1].lower()] = (
+                    line_number, untranslated_file)
+            elif word_list[0] == "let" and (len(word_list) > 1 and word_list[1].startswith("g:menutrans_")):
+                untranslated_dict[word_list[1].lower()] = (
+                    line_number, untranslated_file)
+
+    # PopUp doesn't need to be translated
+    untranslated_dict.pop("popup", None)
+
+    # blank with digit doesn't neet to be translated
```

Comment:
Comment out of date?

We're skipping two cases here:
1. String starting and ending with '\ '.
2. String contains only single digit.